src/relay/backend/te_compiler_cache.h

```diff
@@ -213,6 +213,7 @@ CachedFunc PrimFuncFor(const Function& source_func, const Target& target,
 CachedFunc ShapeFuncFor(const Function& prim_func, const Target& target,
                         std::function<std::string(std::string)> renamer);
 
+// TODO(mbs): Bring name uniqification under control -- this is replicated in quite a few places.
```

Comment:
Great, probably worth adding ticket for all the follow up.

Reply:
done.